bigchaindb/backend/localmongodb/query.py

```diff
@@ -39,3 +40,20 @@ def get_spent(conn, transaction_id, output):
                       {'_id': 0}))
     except IndexError:
         pass
+
+
+@register_query(LocalMongoDBConnection)
+def get_latest_block(conn):
+    return conn.run(
+        conn.collection('blocks')
+        .find_one(sort=[('height', DESCENDING)]))
+
+
+@register_query(LocalMongoDBConnection)
+def create_new_block(conn, block):
```

Comment:
We have other methods called `store_<obj>`, like `store_transaction` or `store_asset`. Following this pattern, we can call this method `store_block`, what do you think?

Reply:
sure